app/concepts/clients/repository.rb

```diff
@@ -0,0 +1,13 @@
+module Clients
+  class Repository < ::Base::BaseRepository
+    attr_reader :adapter
+
+    def initialize(adapter: Client)
+      super(adapter: adapter)
+    end
+
+    def fake_client
+      adapter.where(real_user: false)
```

Comment:
Ok, so the name and usage I saw till now suggests that there will be one fake client and you are always using `first` after `fake_client` - in that case you can use `find_by(real_user: false)` it will return already single record